tests/testthat/test-drive_list.R

```diff
@@ -8,97 +8,133 @@ context("List files")
 # (pesky_files <- drive_list(pattern = "foo|bar|baz|yo"))
 # pesky_files$id %>% purrr::map(drive_file) %>% purrr::map(drive_delete)
 
+nm_ <- function(x) paste0(x, "-TEST-drive-list")
```

Comment:
Oh, I see you're making the slug specific to this file, but also short. You could define a function for making a function that adds a slug in helper.R and then call it in each file. It doesn't really matter. We might start to care if the slugs get more complicated, i.e. indicate who's running the tests.

``` r
## in helper.R
nm_fun <- function(slug) {
  function(x) paste0(x, slug)
}

## at the top of this file
nm_ <- nm_fun("-TEST-drive-list")

## usage
nm_("hi")
#> [1] "hi-TEST-drive-list"
```